data/data-pipeline/data_pipeline/etl/sources/ejscreen/etl.py

```diff
@@ -24,7 +24,6 @@ def __init__(self):
 
         self.COLUMNS_TO_KEEP = [
             self.GEOID_TRACT_FIELD_NAME,
-            field_names.TOTAL_POP_FIELD,
```

Comment:
We were getting total population from EJSCREEN. I looked up the field they use for total pop and directly accessed it from Census. Which helps because now we can use that variable in the census calculations.